streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/serialization/Serializer.java

```diff
@@ -0,0 +1,9 @@
+package org.ray.streaming.runtime.serialization;
+
+public interface Serializer {
```

Comment:
What is this `Serializer` used for? Shall we implement `CrossLangSerializer` in `ray-runtime` module? I can see we "just implement in place to make compiler happy". Though, it can be a follow-up cleanups...

Reply:
Ray call serialization is mainly for control flow. Streaming is mainly for data flow. These distinctions make the design and implementation different. Currently we just put a simple implementation here. But a more sophisticated and feature-complete serialization framework will be submitted in later PR.